client/tailwind.config.ts

```diff
@@ -87,6 +87,15 @@ export default {
             fontWeight: "400"
           }
         ]
+      },
+      screens: {
+        xxs: [{ min: "320px" }],
```

Comment:
:+1: will be helpful in future

Reply:
normally I think tailwind has them as default, but in case we need to change in future this is fine

Reply:
Yup I kept the rest as default values but just added them in case later on.